src/lib/Form/Processor/Content/AutosaveProcessor.php

```diff
@@ -35,9 +36,12 @@ public static function getSubscribedEvents(): array
 
     public function processAutosave(FormActionEvent $event): void
     {
+        $statusCode = Response::HTTP_OK;
+
         try {
-            $this->innerContentFormProcessor->processSaveDraft($event);
-            $statusCode = Response::HTTP_OK;
+            if ($event->getData() instanceof ContentUpdateData) {
```

Comment:
Is this a reliable way to detect `/nodraft` requests? Can we have some test coverage to be sure that we didn't disabled autosave for cases that shouldn't be affected by this change?

Reply:
@konradoboza I can create some tests, but what other cases do you have in mind that shouldn't be affected by this change? In `processSaveDraft` we explicitly mention only two cases - `ContentCreateData` and `ContentUpdateData` (https://github.com/ezsystems/ezplatform-content-forms/blob/1.3/src/lib/Form/Processor/ContentFormProcessor.php#L83). It's even more reasonable to block other unexpected `Datas` here and leave only the one fitting.

Reply:
I assume that with this change we don't block autosaving while being on content creation form, right? The naming itself is a bit hard to wrap a head around without context. That's why I asked about some test coverage. Nevertheless, if it isn't the case, I am fine with proposed changes.

Reply:
>If by content creation form you mean creation of content that is in fact updating its first draft

That was indeed my assumption, but wasn't sure.

Reply:
Just adding to the above comment:
clicking on "+ Create" in Back Office creates a draft and then redirects us to updating such a draft.

Reply:
It seems, that this form has `drafts_enabled` option, would that be a better way to check it? It just seems it is set wrong for /nodraft action and it is only used in UDW.

Reply:
I guess best would be to make this:  https://github.com/ezsystems/ezplatform-content-forms/blob/1.3/src/lib/Content/View/Filter/ContentCreateViewFilter.php#L132 optional parameter and set it to `false` here: https://github.com/ezsystems/ezplatform-content-forms/blob/1.3/src/lib/Content/View/Filter/ContentCreateViewFilter.php#L87

This should give us reliable information whether drafts are enabled or not.

Reply:
yup, for me form option sounds better then `instance of` check.
On the other hand, if it only breaks displaying of buttons, then maybe it is ok. If people wants to have `Save draft` button, when using /nodraft path, that contradicts a little to each other but I guess thats another discussion and I am all for another option.

Reply:
@ViniTou @Nattfarinn Closing this in favor of https://github.com/ezsystems/ezplatform-content-forms/pull/66